salt/modules/x509.py

```diff
@@ -1388,19 +1388,29 @@ def create_certificate(
         for ignore in list(_STATE_INTERNAL_KEYWORDS) + ['listen_in', 'preqrequired', '__prerequired__']:
             kwargs.pop(ignore, None)
 
-        certs = __salt__['publish.publish'](
-            tgt=ca_server,
-            fun='x509.sign_remote_certificate',
-            arg=six.text_type(kwargs))
+        ca_servers = []
+        if type(ca_server) is list:
+            ca_servers = ca_server
+            random.shuffle(ca_servers)
+        else:
+            ca_servers.append(ca_server)
+        for server in ca_servers:
```

Comment:
For this, can we instead do

```
if not isinstance(ca_server, list):
    ca_server = [ca_server]
random.shuffle(ca_server)
for server in ca_server:
```

Just to simplify it a bit, and remove extra assignments.

Reply:
Applied